### PR TITLE
[ACTION] Giphy new actions 

### PR DESCRIPTION
**Pull request by vellames-turing**
New Actions

- [x] Search Gifs/Stickers | Searches all GIPHY gifs or stickers for a word or phrase
- [x] Translate a Word to a Gif/Sticker | Translates or convert a word or phrase to the perfect Gif or Sticker using GIPHY's special sauce algorithm
- [x] Upload a Gif | Uploads a Gif or video file up to 100MB programmatically on GIPHY.com

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/giphy/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "description": "Pipedream Giphy Components",
+    "main": "giphy.app.js",
+    "keywords": [
+        "pipedream",
+        "giphy"
+    ],
+    "homepage": "https://pipedream.com/apps/giphy",
+    "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
+    "license": "MIT",
+    "publishConfig": {
```
Comment (jcortes) [inline components/giphy/package.json:13]:
```suggestion
    "license": "MIT",
    "gitHead": "e12480b94cc03bed4808ebc6b13e7fdb3a1ba535",
    "publishConfig": {
```

**Review by jcortes — CHANGES_REQUESTED** [review]
It looks good to me. Just a few minor request changes and it will be ready for QA!

**Review by jcortes — APPROVED** [review]
@vellames-turing It looks good to me! Ready for QA

**Comment by dmathukorala** [comment]
I have verified the changes and it works as expected without any issues. This is ready to merge.

**Pull request merged**